### PR TITLE
[iOS10, UIKit] More work

### PR DESCRIPTION
**Pull request by migueldeicaza**

### CODE REVIEW

File: src/uikit.cs [hunk 1]
```diff
@@ -15494,6 +15569,58 @@ public interface UIPressesEvent {
...
+	
+		[Export ("cancelInteraction")]
+		void CancelInteraction ();
+	}
+	
+	[Protocol]
+	[BaseType (typeof(NSObject))]
+	interface UIPreviewInteractionDelegate
+	{
+		[NoWatch, NoTV, iOS (10,0)]
```
Comment (spouliot) [inline src/uikit.cs]: the attributes should be moved to the type level (all identical)

File: src/uikit.cs [hunk 2]
```diff
@@ -15494,6 +15569,58 @@ public interface UIPressesEvent {
...
+		[Export ("previewInteractionDidCancel:")]
+		void DidCancel (UIPreviewInteraction previewInteraction);
+	
+		[NoWatch, NoTV, iOS (10,0)]
+		[Export ("previewInteractionShouldBegin:")]
+		[DelegateName ("Func<UIPreviewInteraction,bool>"), DefaultValue(true)]
+		bool ShouldBegin (UIPreviewInteraction previewInteraction);
+	
+		[NoWatch, NoTV, iOS (10,0)]
+		[Export ("previewInteraction:didUpdateCommitTransition:ended:")]
```
Comment (spouliot) [inline src/uikit.cs]: that won't compile ;-)

File: src/uikit.cs [hunk 3]
```diff
@@ -15494,6 +15569,58 @@ public interface UIPressesEvent {
...
+	
+		[Wrap ("WeakDelegate")]
+		[NullAllowed]
+		UIPreviewInteractionDelegate Delegate { get; set; }
+	
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		NSObject WeakDelegate { get; set; }
+	
+		[Export ("locationInCoordinateSpace:")]
+		CGPoint LocationInCoordinateSpace ([NullAllowed] UICoordinateSpace coordinateSpace);
```
Comment (spouliot) [inline src/uikit.cs]: prefix with `Get`

**Comment by xamarin-release-manager** [comment]
Build failure

**Comment by spouliot** [comment]
@dalexsoto since @migueldeicaza is busy would you mind continuing this PR and completing the bindings for UIKit ? thanks!

**Comment by dalexsoto** [comment]
Now this is tracked on https://github.com/xamarin/xamarin-macios/pull/566

**Pull request closed without merging**